### DISCUSSION TITLE
SDLC Member Feedback: PR to resolve #2024 Security queries are not implemented to spec

### DISCUSSION THREAD

**Discussion by theresalech** [smartdevicelink/sdl_ios#2030] [General]
The SDLC review period for PR #2026 starts now and continues through September 21, 2021. Following the review period, the Project Maintainer will conduct final testing to prepare the PR for release in iOS 7.3.

Please leave any any questions, comments, or other feedback about PR #2026 and/or Issue #2024  here, so that necessary discussion can take place as soon as possible.

Thank you!

**Comment by mrapitis** [comment]
We have integrated PR #2026 into a sample app and have performed testing, results have been positive thus far.

**Comment by theresalech** [comment]
The SDLC review period has concluded for this PR. It has been merged and will be released in iOS 7.3 on October 27, 2021. Thank you for testing!
